src/rpm_ostree/actor.rs

```diff
@@ -53,7 +53,9 @@ impl Handler<StageDeployment> for RpmOstreeClient {
 
     fn handle(&mut self, msg: StageDeployment, _ctx: &mut Self::Context) -> Self::Result {
         trace!("request to stage release: {:?}", msg.release);
-        super::cli_deploy::deploy_locked(msg.release, msg.allow_downgrade)
+        let release = super::cli_deploy::deploy_locked(msg.release, msg.allow_downgrade);
+        trace!("rpm-ostree CLI returned: {:?}", release);
```

Comment:
It may be good at some point to investigate using the [tracing crate](https://crates.io/crates/tracing) for zincati.  It's much better for async in particular.

Hmm wait, isn't something logging this `Result` if it's not `Ok` somewhere already?

Reply:
There is currently logging at the `update_agent` actor side. I was mainly trying to figure out whether rpm-ostree CLI returned at all, or whether messages aren't being sent properly across actors, since I wasn't able to tell from the logs.